examples/token/run.sh

```diff
@@ -0,0 +1,240 @@
+#!/bin/bash
+# First check that Leo is installed.
+if ! command -v leo &> /dev/null
+then
+    echo "leo is not installed."
+    exit
+fi
+
+# The private key and address of the Alice.
+# Swap these into program.json, when running transactions as the first bidder.
+# "private_key": "APrivateKey1zkp1w8PTxrRgGfAtfKUSq43iQyVbdQHfhGbiNPEg2LVSEXR",
+# "address": "aleo13ssze66adjjkt795z9u5wpq8h6kn0y2657726h4h3e3wfnez4vqsm3008q"
+
+# The private key and address of the Bob.
+# Swap these into program.json, when running transactions as the second bidder.
+# "private_key": "APrivateKey1zkpFo72g7N9iFt3JzzeG8CqsS5doAiXyFvNCgk2oHvjRCzF"
+# "address": "aleo17vy26rpdhqx4598y5gp7nvaa9rk7tnvl6ufhvvf4calsrrqdaqyshdsf5z"
+
+# Swap in the private key and address of the Alice to .env.
```

Comment:
```suggestion
# Swap in the private key and address of Alice to .env.
```

Reply:
This is just the private key, not the address.  Do we need to add the address or change the comment?
(There are others like this that I will not repeat.)